strawberryfields/apps/qchem/dynamics.py

```diff
@@ -72,7 +72,12 @@
 
 - The function :func:`~.prob` computes the probability of observing a desired excitation in the
   generated samples.
+
+- The function :func:`~.sample_coherent` generates samples for simulating vibrational quantum
+  dynamics in molecules with a coherent input state.
```

Comment:
I'm not sure if these bullets are helping much more than what's already provided in the summary boxes:

![image](https://user-images.githubusercontent.com/49409390/85600887-dc9efa80-b61b-11ea-9f37-deabb08e61be.png)

Summary boxes:

![image](https://user-images.githubusercontent.com/49409390/85601266-356e9300-b61c-11ea-8cb0-3a13bb56fd9c.png)

When we finish adding all the features for this module, we should revisit turning these into a paragraph or two in a more pedagogical style.